evap/evaluation/templatetags/evaluation_filters.py

```diff
@@ -221,3 +221,10 @@ def order_by(iterable, attribute):
 @register.filter
 def get(dictionary, key):
     return dictionary.get(key)
+
+
+@register.filter(name="add_class")
```

Comment:
Since the name is the same as the function, you can use:
```suggestion
@register.filter
```